src/app/store.service.ts

```diff
@@ -18,9 +28,82 @@ export class StoreService {
   constructor(
     private packageService: PackageService,
     private itemService: ItemService,
-    private authService: AuthService
+    private authService: AuthService,
+    private http: HttpClient
   ) { }
 
+  getAddressFromBackend(backendAddress: string): Address {
+    const splitAddr = backendAddress.split('\n')
+      return {
+        addressLine1: splitAddr[0],
+        addressLine2: splitAddr[1],
+        postalCode: splitAddr[2]
+    }
+  }
+
+  mockLogistics(): Logistics {
+    return {
+      partner: {
+        id: 'ninjavan'
+      },
+      pickUpAddress: {
+        addressLine1: '',
+        addressLine2: '',
+        postalCode: '',
+        useBusinessAddress: true
+      }
+    }
+  }
+
+  mockBusiness(): Business {
+    return {
+      name: 'Not Starbucks',
+      description: 'This store is not Starbucks.',
+      type: 'Food and Beverage'
+    }
+  }
+
+  async mapPackageItemToStorePackageItem(packageItem: PackageItem, merchantId: string): Promise<StorePackageItem> {
+    return {
+      item: await this.itemService.getItem(merchantId, packageItem.itemId),
+      quantity: packageItem.quantity
+    }
+  }
+
+  async mapPackageToStorePackage(inputPackage: Package, backEndStore: BackEndStore): Promise<StorePackage> {
+    return {
+      id: inputPackage.id,
+      merchantId: inputPackage.merchantId,
+      merchantName: backEndStore !== null ? backEndStore.name : this.mockBusiness.name,
```

Comment:
Do we need to have mocked business object? We can get from backend right? Using the merchant endpoint

Reply:
if they don't have business yet then what should we return?

Reply:
I see what you mean... Technically all merchants should have this value. Cuz it's sort of a required part of the form during registration. May be safe to assume this will always be there

business: {
    name: '',
    type: undefined,
    description: ''
  }